pkg/reconciler/deployment/controller.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package deployment
+
+import (
+	"context"
+	"github.com/google/knative-gcp/pkg/reconciler"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/cache"
+	"knative.dev/pkg/client/injection/kube/informers/apps/v1/deployment"
+	"knative.dev/pkg/client/injection/kube/informers/core/v1/secret"
+	"knative.dev/pkg/configmap"
+	"knative.dev/pkg/controller"
+)
+
+const (
+	// ReconcilerName is the name of the reconciler
+	ReconcilerName = "Deployment"
+
+	// controllerAgentName is the string used by this controller to identify
+	// itself when creating events.
+	controllerAgentName = "cloud-run-events-deployment-controller"
+
+	namespace      = "cloud-run-events"
+	secretName     = "google-cloud-key"
```

Comment:
there is already a constant in pkg/apis/duck/v1alpha1 google_cloud_secret_defaults... It's private, can you rename to DefaultSecretName, and maybe point it here... secretName = DefaultSecretName....
so that we don't mess up if we update it later on